app/ports/tests/sample_data/portindex2json.tcl

```diff
@@ -2,8 +2,11 @@
 # a list of JSON objects.
 # Written by Joshua Root <jmr@macports.org>, 2017
 # Requires: tclsh with the tcllib 'json' package.
-# Usage: tclsh portindex2json.tcl path/to/PortIndex
-
```

Comment:
Oh, wait! Couldn't/shouldn't we clone the script from the upstream repository instead? It just feels wrong to keep a literally copy here.

Reply:
Okay, thank you.